### PR TITLE
docs: [RU] Translation update

### PR DESCRIPTION
**Pull request by Alex-Sokolov**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Underlying tools
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

**Other information:**
Latest RU docs translation update

### CODE REVIEW

File: docs/ru/core-plugins/pwa.md [hunk 1]
```diff
@@ -106,6 +106,12 @@
                   }
                 ]`
 
+- **pwa.manifestCrossorigin**
+
+  - По умолчанию:: `undefined`
```
Comment (NataliaTepluhina) [inline docs/ru/core-plugins/pwa.md:111]:
```suggestion
  - По умолчанию: `undefined`
```

File: docs/ru/core-plugins/pwa.md [hunk 2]
```diff
@@ -106,6 +106,12 @@
...
+
+    Значение атрибута `crossorigin` в теге ссылки на манифест в сгенерированном HTML. Это значение может потребоваться установить, когда PWA находится за аутентифицированном прокси-сервером. Для более подробной информации смотрите [варианты значений cross-origin](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#attr-crossorigin).
```
Comment (NataliaTepluhina) [inline docs/ru/core-plugins/pwa.md:113]:
```suggestion
    Значение атрибута `crossorigin` в теге ссылки на манифест в сгенерированном HTML. Это значение может потребоваться установить, когда PWA находится за аутентифицированным прокси-сервером. Для более подробной информации смотрите [варианты значений cross-origin](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#attr-crossorigin).
```

**Comment by Alex-Sokolov** [comment]
@NataliaTepluhina thanks :) fixed

**Comment by NataliaTepluhina** [comment]
@Alex-Sokolov thanks for the blazing fast fix 😅  Merging!

**Pull request merged**